src/v2/Apps/ViewingRoom/Components/ViewingRoomClosed.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import { Box, Button, Flex, Sans } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ViewingRoomClosed_viewingRoom } from "v2/__generated__/ViewingRoomClosed_viewingRoom.graphql"
+
+interface ViewingRoomClosedProps {
+  viewingRoom: ViewingRoomClosed_viewingRoom
+}
+
+const ViewingRoomClosed: React.FC<ViewingRoomClosedProps> = props => {
+  const {
+    viewingRoom: {
+      partner: { href },
+    },
+  } = props
+
+  const handleViewGalleryClick = () => {
+    window.location.assign(href)
+  }
+
+  return (
+    <Box>
+      <Flex flexDirection="column" alignItems="center">
+        <Sans size="4" mt={3} mb={1} textAlign="center" maxWidth="470px">
+          This viewing room is now closed. We invite you to view this gallery's
+          current works.
+        </Sans>
+        {href && (
+          <Box>
+            <Button variant="secondaryGray" onClick={handleViewGalleryClick}>
```

Comment:
We should always try to avoid links that aren't a tags, or rather -- if a link _can_ be an a tag thats much better. 

So like 
```
<RouterLink to={...}>
  <Button>... 
```

Reply:
Thanks! Did not know about `RouterLink`. Updated.